# An array of particles with a header for global reference.

std_msgs/Header header

int32           id
Particle[]      particles
